.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+dist/
+.env
+.vscode/
```

Comment:
hi Thanks, 
Files should always end with line breaks. This tends to create problems between Operatins systems sometimes, that is why is pointed out.